packages/app/src/app/pages/common/Modals/EmptyTrash/index.tsx

```diff
@@ -0,0 +1,30 @@
+import { Alert } from 'app/components/Alert';
```

Comment:
This `Alert` component expects a "string" on the property "body". It would be great if you would go to this component and change out this interface:

```ts
interface IAlertProps {
  title: string;
  body: string;
  confirmMessage?: string;
  onCancel: (event: React.MouseEvent<HTMLButtonElement>) => void;
  onConfirm: (event: React.MouseEvent<HTMLButtonElement>) => void;
}
```

To:

```ts
import React, { ReactNode } from 'react'

interface IAlertProps {
  title: string;
  body: string | ReactNode;
  confirmMessage?: string;
  onCancel: (event: React.MouseEvent<HTMLButtonElement>) => void;
  onConfirm: (event: React.MouseEvent<HTMLButtonElement>) => void;
}
```